pom.xml

```diff
@@ -190,6 +186,19 @@
 				<version>${jodatime}</version>
 				<scope>test</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>org.apache.commons</groupId>
```

Comment:
Do we really need these new dependencies? I am especially worried about the Commons Lang version as it's very likely to be an application dependency as well so that version conflicts are likely. We should try to avoid additional dependencies if possible.